### ISSUE TITLE
smartthings Device Network Id reverts to mac address

### ISSUE THREAD

**Issue by Mark55347** [BobRak/OpenHAB-Smartthings#42]
I am finding when I go into the smartthings developers site that the shortly after changing the Device Network Id (Mac Address without ":" ). It reverts back to the MAC ID (It appears to be with dashes instead of quotes). This prevents the discovery process from working.  Discovery only works in that brief interim time before the reversion occurs.

It is possible as I am fairly new to this, that I have somehow created this issue, but for the life of me I do not know how.

Mark

My bad. I found the reversion is coming from the MAC setting on my smartthings hub.
I am leaving the issue open, as the updated one on the developers website should have been passed down to the hub.  This, however may be beyond your control, so feel free to close it out.
You may however, want to change it from a MAC setting to network identifier name, or some such.

**Comment by BobRak** [comment]
Mark:  Thanks for the feedback. I'm going to leave it open for now. Right now I'm in the process of moving and it has been dragging on longer that I anticipated it would. Currently my PC is in one state and my Smartthings hub, RPi, etc is a state away. Once I finish my move I'll see what is happening on my setup. Unfortunately that could be as late as early August.
Bob

**Comment by Mark55347** [comment]
Thanks for the update.
Once I figured out what was happening I have had no problems.
Have fun with your move.
Mark
--------------------------------------------
On Mon, 6/11/18, BobRak <notifications@github.com> wrote:

 Subject: Re: [BobRak/OpenHAB-Smartthings] smartthings Device Network Id reverts to mac address (#42)
 To: "BobRak/OpenHAB-Smartthings" <OpenHAB-Smartthings@noreply.github.com>
 Cc: "Mark55347" <markming55347@yahoo.com>, "Author" <author@noreply.github.com>
 Date: Monday, June 11, 2018, 12:03 PM
 
 Mark:  Thanks for the feedback.
 I'm going to leave it open for now. Right now I'm in
 the process of moving and it has been dragging on longer
 that I anticipated it would. Currently my PC is in one state
 and my Smartthings hub, RPi, etc is a state away. Once I
 finish my move I'll see what is happening on my setup.
 Unfortunately that could be as late as early August.
 
 Bob
 
 —
 You are
 receiving this because you authored the thread.
 Reply
 to this email directly, view
 it on GitHub, or mute
 the thread.
